README.md

```diff
@@ -26,20 +26,21 @@ to develop a new feature.
 
 Find an existing issue (e.g. one marked [starter], or simply
 ask an [owner](https://github.com/kubernetes/helm/blob/master/OWNERS) for suggestions), and respond on the
-issue thread expressing interest in working on it. 
- 
+issue thread expressing interest in working on it.
+
 This helps other people know that the issue is active, and
 hopefully prevents duplicated efforts.
 
-Before submitting a pull request, sign the [CLA].
+Each commit must be signed in git, as described by
```

Comment:
signed off? just to avoid confusing with https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work